packages/rancher-sriov/generated-changes/overlay/templates/secrets.yaml

```
@@ -0,0 +1,20 @@
+{{- if not .Values.cert_manager -}}
+{{- if .Values.operator.enableAdmissionController }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: operator-webhook-service
+  namespace: {{ .Release.Namespace }}
+data: {{ include "sriov_operator_cert" . | indent 2 }}
```

need the newline right?
```suggestion
data: {{ include "sriov_operator_cert" . | nindent 2 }}
```